x-pack/plugin/core/src/main/java/org/elasticsearch/index/engine/frozen/FrozenEngine.java

```diff
@@ -257,6 +257,7 @@ private Engine.Searcher openSearcher(String source, SearcherScope scope) throws
                 assert false : "refresh_needed is always false";
             case "segments":
             case "segments_stats":
+            case "ignored_field":
```

Comment:
@martijnvg I added this entry which, if I understand correctly results in not opening a new searcher but just reusing one that is already open.